dienynas.php

```diff
@@ -16,24 +16,24 @@
 }
 
 $saved = "";
+$studentDataChunks = "";
 $studentNumber = "";
-$studentName1 = "";
-$studentName2 = "";
+$studentName = "";
 $studentSurname = "";
 if (isset($_POST['student']) and isset($_POST['subject']) and isset($_POST['mark']) and isset($_POST['notes'])) {
     $studentsFile = fopen($studentsFilename, "r");
     if ($studentsFile !== FALSE) {
-        while (($name = fgetcsv($studentsFile, ",")) !== FALSE) {
-            if (in_array($_POST['student'], $name)) {
-                $studentName1 = $name[0];
-                $studentName2 = $name[1];
-                $studentSurname = $name[2];
-                $studentNumber = $_POST['student'];
+        while (($studentData = fgetcsv($studentsFile, ",")) !== FALSE) {
+            if (in_array($_POST['student'], $studentData)) {
+                $studentData[2] = str_replace("  ", ", ", $studentData[2]);
```

Comment:
Kas čia per pakeitimai vyksta? Papasakok ką darai ir kam to reikia?

Reply:
čia jau nebėra nei antro vardo kintamojo nei str_replace funkcijos. 
$studentName = $studentData [2];
$studentSurname = $studentData[1];
$studentNumber = $studentData[0];

Reply:
tai šitie kintamieji  aiškiau išskiria, kuris aibės studentData elementas yra vardas, kuris pavardė, kuris numeris.